app/assets/javascripts/application.js

```diff
@@ -255,7 +255,7 @@ function template_info(div, url) {
   // Use a post to avoid request URI too large issues with big forms
   $.ajax({
     type: "POST",
-    url: url + "&provisioning=" + build,
+    url: url + ((url.indexOf('?') == -1) ? '?' : '&') + "provisioning=" + build,
```

Comment:
nitpick: isnt it enough to find the first usage of ? vs anywhere in the string? for example:
```javascript
(new URL(url)).search[0] === '?'
```